frontend/src/metabase/components/LastEditInfoLabel/LastEditInfoLabel.unit.spec.js

```diff
@@ -20,7 +20,10 @@ describe("LastEditInfoLabel", () => {
     email: "john@metabase.test",
   });
 
-  function setup({ isLastEditedByCurrentUser = false } = {}) {
+  function setup({
+    isLastEditedByCurrentUser = false,
+    onClick = jest.fn(),
```

Comment:
`LastEditInfo` were expected to be a button whether or not it's passed an `onClick` prop.

We don't pass `onClick` on 2 instances:
1. In search results
    ![Screenshot 2024-10-15 at 4 20 11 PM](https://github.com/user-attachments/assets/4575f6e3-aa6e-4632-a1cb-43fda8d37dcb)

3. On SDK

So, I default the behavior to have `onClick` value.